crates/rapid-web/src/tui.rs

```diff
@@ -1,40 +1,14 @@
 use colorful::{Color, Colorful};
-use rapid_cli::tui::rapid_logo;
 use std::env;
-use tiny_gradient::{GradientDisplay, GradientStr, RGB};
-
-pub fn started_server_message() -> colorful::core::color_string::CString {
-	"Started".bg_blue().color(Color::White).bold()
-}
+use log::info;
 
 pub fn server_init(bind_config: (String, u16)) {
-	// This print statement removes all prev terminal logs (aka the same as running a clear command)
-	print!("{esc}c", esc = 27 as char);
 	let server_url = format!("http://{}:{}", &bind_config.0, &bind_config.1).color(Color::Blue);
 	// Grab the current crate version at compile time...
 	let crate_version = env!("CARGO_PKG_VERSION");
-	println!(
-		"{} {} {} and serving requests: \n\n➜  {} {}",
-		rapid_logo(),
-		crate_version.bold(),
-		started_server_message(),
-		"Dev Server:".bold(),
+	info!(
+		"started rapid version {} server on {}",
```

Comment:
My inspiration for this was to make things similar to how Vite does it (leaving the rapid_logo would be nice):
<img width="454" alt="Screenshot 2023-08-01 at 9 55 17 AM" src="https://github.com/Cincinnati-Ventures/rapid/assets/68653294/d9044037-3cbb-4145-9796-167ab7a0e254">